packages/taco/src/taco.ts

```diff
@@ -140,8 +140,8 @@ export const decrypt = async (
   provider: ethers.providers.Provider,
   domain: Domain,
   messageKit: ThresholdMessageKit,
+  authProvider: EIP4361AuthProvider,
```

Comment:
Should this be optional or is it always provided? It's not needed if the condition doesn't actually use the user address context variable.

Reply:
Right, that makes sense